src/state/state-persister.js

```diff
@@ -0,0 +1,32 @@
+const fs = require('fs')
+const os = require('os')
+const path = require('path')
+
+const mobTimerDir = path.join(os.homedir(), '.mob-timer')
```

Comment:
Great change ❤️

Reply:
Parker really gets the credit for that one. I would have just moved the file to the home dir, but making it a directory (where we could possibly put other stuff in the future) just seemed better.